packages/vanilla/src/components/footer/footer.stories.js

```diff
@@ -10,10 +10,10 @@ const InternalTemplate = () => {
     <footer class="cbp-footer">
       <nav>
         <ul>
-          <li><a href="#">App Overview</a></li>
-          <li><a href="#">Trainings</a></li>
-          <li><a href="#">FAQ's</a></li>
-          <li><a href="#">Release Notes</a></li>
+          <li><a href="#" class="cbp-btn">App Overview</a></li>
+          <li><a href="#" class="cbp-btn">Trainings</a></li>
+          <li><a href="#" class="cbp-btn">FAQ's</a></li>
+          <li><a href="#" class="cbp-btn">Release Notes</a></li>
```

Comment:
I don't think we should put the `cbp-btn` classes here since they don't look or act like the buttons we have. We're deleting 3-4 css properties by placing the class here and I think it's a little more confusing to place the class in here. I think the only thing that needed fixing was the `letter-spacing` to match the buttons.

Reply:
I'll revert it. all those deleted CSS props are part of button, so I feel like it's 6 one way, half a dozen the other. Seems arbitrary we're styling links like buttons.